e2e/e2e.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```

Comment:
Please feel free to readd the [e2e/clean_db.py](https://github.com/ElektraInitiative/PermaplanT/pull/784/files#diff-44b41e8fd775fbf87820e00096a71208d08866206f953bec7fcdaed861c8ee03).